### ISSUE TITLE
Website for replacement suggestions

### ISSUE THREAD

**Issue by Lehoczky** [es-tooling/module-replacements#154]
I think it would be a good idea to use the markdown files from the `/docs` folder and compile them into website with a tool like [vitepress](https://vitepress.dev/) or [starlight](https://starlight.astro.build/).

## Use case

If someone just quickly wants to look up whether a dependency has other alternatives, a website with search functionality can be the easiest.

**Comment by 43081j** [comment]
I agree, would love to have a website for this

We could possibly also combine it with some web tooling, e.g. visualisation of replaceable modules and available codemods (have been discussing some of this in discord) 

Maybe we could use gh pages?

**Comment by Lehoczky** [comment]
Yeah, GH pages would be great: it's free and it's easy to deploy to.

**Comment by Namchee** [comment]
I'm personally against creating a new website for this. Why can't we just create a new section in [e18e](https://e18e.dev/)?

**Comment by Joery-M** [comment]
As a fun side project, I made https://module-replacements.dev/, it just takes the manifests and markdown to visualize them, nothing special. Completely open to adding features or even to hand over the domain.

[Example](https://module-replacements.dev/?q=documented-body-parser)
[Github](https://github.com/Joery-M/module-replacements-website)
